src/hardware/rdtech-dps/protocol.c

```diff
@@ -85,6 +85,8 @@ enum rdtech_rd_register {
 	REG_RD_PROTECT = 16, /* u16 */
 	REG_RD_REGULATION = 17, /* u16 */
 	REG_RD_ENABLE = 18, /* u16 */
+	REG_RD_PRESET = 19, /* u16 */
+	REG_RD_RANGE = 20, /* u16 */
```

Comment:
I'd like a note that says whether these numeric values are arbitrary or part of the protocol. In contrast to the 'state mask' enum that appears to be arbitrary and only for maintaining local state ?

Reply:
They are not arbitrary. These are the device family specific modbus registers. I think that you have a point that this should be stated in a comment. I'll do that.